kitsune/search/v2/documents.py

```diff
@@ -47,6 +48,19 @@ def update_document(cls):
     @classmethod
     def prepare(cls, instance):
         """Override super method to merge docs for KB."""
+        # Add a discard field in the document if the following conditions are met
+        # Wiki document is a redirect
+        # Wiki document is archived
+        # Wiki document is a template
+        if any(
+            [
+                instance.html.startswith(REDIRECT_HTML),
+                instance.is_archived,
```

Comment:
This is currently a localeaware field, which means at least at one point we thought this could differ among locales. If we don't think that's the case any more, we should remove the `is_archived` field from the mapping - though we should check this with Joni first.

Reply:
regardless, I will update this PR to check the flag for each specific locale. If this turns out to be the same for all children we can make the field non locale aware in our index.